contrib/haproxy/template.py

```diff
@@ -0,0 +1,42 @@
+pkgname = "haproxy"
+pkgver = "2.9.3"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_args = [
+    "LUA_LIB=/usr/lib/lua5.4",
+    "LUA_LIB=/usr/include/lua5.4",
```

Comment:
both of these are autodetected from what i can tell

Reply:
ok